### PR TITLE
Hide 'Copy to clipboard' on non-code blocks and improvements to `Getting Started` section

### PR DESCRIPTION
**Pull request by black-dragon74**
[QOL] Hide 'Copy To Clipboard' button on non-code blocks
Site wide there are multiple code blocks that do not contain
any code in them. But still a `Copy code to clipboard` button
is shown, this patch addresses this issue. It is a QOL improvement
that makes distinguishing code blocks visually and functionally easier.

To use this new feature, we need to start our code block like:
```text
```{ .language_name .no-copy }
```

Where `language_name` is something that can vary (`console` in our case mostly).
The `.no-copy` attribute is what disables the function on that particular block.

This is a first of many patches that'll follow for subsequent sections site-wide
which moves OLD `md` syntax to NEW one and also address any other semantics that come
into notice.

Regards

### CODE REVIEW

**Comment by black-dragon74** [comment]
CC: @aravindavk @kshithijiyer @Shwetha-Acharya 

Regards

**Comment by black-dragon74** [comment]
You can see how the copy code to clipboard is not visible in the last code block below:

Screen Shot 2022-05-13 at 1 59 21 PM

Regards

File: docs/Quick-Start-Guide/Architecture.md [hunk 1]
```diff
@@ -122,17 +135,20 @@ without interrupting the operation of the volume.
 Create a dispersed volume:
 
 ```
```
[the remark this answers is not in the document]
Comment (black-dragon74) [reply docs/Quick-Start-Guide/Architecture.md:137]: Nice catch, totally missed it as it was not present there already :(

Fixing all occurrences.

Regards

File: docs/Quick-Start-Guide/Architecture.md [hunk 2]
```diff
@@ -17,10 +17,10 @@ Gluster file system supports different
 types of volumes based on the requirements. Some volumes are good for
 scaling storage size, some for improving performance and some for both.
 
-1.__ **Distributed Glusterfs Volume** __- This is the type of volume which is created by default if no volume type is specified.
+1.\*\* **Distributed Glusterfs Volume** **- This is the type of volume which is created by default if no volume type is specified.
```
Comment (Shwetha-Acharya) [inline docs/Quick-Start-Guide/Architecture.md:20]: This would appear as ** Distributed Dispersed Glusterfs Volume**
Not sure if ** are required besides

Comment (black-dragon74) [reply docs/Quick-Start-Guide/Architecture.md]: For some reason, prettier thought it would be cool to add them :P

Fixed it everywhere.

Regards

**Review by Shwetha-Acharya — APPROVED** [review]

**Pull request merged**